apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,10 +1,31 @@
+import { NavLink } from "react-router-dom";
+
 type Properties = {
+	className?: string | undefined;
+	href?: string | undefined;
 	label: string;
 	type?: "button" | "submit";
 };
 
-const Button = ({ label, type = "button" }: Properties): JSX.Element => (
-	<button type={type}>{label}</button>
-);
+const Button = ({
+	className = "",
```

Comment:
This style is the default one for button, so it should be applied here, not passed from the outside 
```suggestion
```

Reply:
i added the styles in the button